website/app/styles/components/do-dont.scss

```diff
@@ -5,12 +5,14 @@
 
 .doc-do-dont {
   margin: 16px 0;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
 }
 
 .doc-do-dont__badge {
   @include doc-font-style-body-small();
   width: min-content;
-  margin-bottom: 8px;
```

Comment:
This might be personal preference, but I prefer to use flexbox whenever possible instead of margin. It accomplishes the same thing visually, but this seems more semantic and potentially more scalable (if anything is added in between or at the end in future).

If margin is preferred, happy to change it back though, especially if that's the spacing strategy in other components.